hnn_core/viz.py

```diff
@@ -553,16 +553,16 @@ def plot_cell_morphology(cell, ax, show=True):
         plt.figure()
         ax = plt.axes(projection='3d')
 
-    ax.set_ylim((-100, 1200))
-    ax.set_xlim((-250, 150))
+    ax.set_xlim((cell.pos[1] - 250, cell.pos[1] + 150))
+    ax.set_ylim((cell.pos[2] - 100, cell.pos[2] + 1200))
     for sec_name, p_sec in cell.p_secs.items():
         linewidth = _linewidth_from_data_units(ax, p_sec['diam'])
         sec_pts = p_sec['sec_pts']
-        xs = [pt[0] for pt in sec_pts]
-        ys = [pt[1] for pt in sec_pts]
-        zs = [pt[2] for pt in sec_pts]
+        xs = [pt[0] + cell.pos[0] for pt in sec_pts]
+        ys = [pt[2] + cell.pos[1] for pt in sec_pts]
+        zs = [pt[1] + cell.pos[2] for pt in sec_pts]
         ax.plot(xs, ys, zs, 'b-', linewidth=linewidth)
-    ax.view_init(90, -90)
+    ax.view_init(0, -90)
```

Comment:
this verifies that [elevation](https://matplotlib.org/2.0.2/mpl_toolkits/mplot3d/api.html#mpl_toolkits.mplot3d.axes3d.Axes3D.view_init) is no longer required ... meaning `z` axis is along which the cell is oriented, not `y`